packages/components/src/components/mobile-full-page-modal/mobile-full-page-modal.tsx

```diff
@@ -85,9 +85,7 @@ const MobileFullPageModal = ({
                         )}
                         {renderPageHeader && renderPageHeader()}
                         <div className='dc-mobile-full-page-modal__header-text'>
-                            {renderPageHeaderText ? (
-                                renderPageHeaderText()
-                            ) : (
+                            {renderPageHeaderElement ?? (
                                 <Text as='p' color='prominent' line_height='m' size='s' weight='bold'>
                                     {page_header_text}
```

Comment:
how we are sure that this is not undefined?

Reply:
well the expression will evaluate to an empty JSX element (<></>) if renderPageHeaderElement is null or undefined. Therefore, we can be sure that the expression will not be undefined 👍